### DISCUSSION TITLE
Add FEATURE_SETTING_STOCK_COUNT_OPENED_PRODUCTS_AGAINST_MINIMUM_STOCK_AMOUNT to configurable features

### DISCUSSION THREAD

**Discussion by Kackerlacka** [hassio-addons/addon-grocy#147] [Ideas]
# Problem/Motivation

> I would like to be able to have it so opening a product in Grocy does not count against the minimum stock amount. This setting is available in config-dist.php, but not one of the configurable options for the HA addon.

## Expected behavior

> Should be able to configure this option.

## Actual behavior

> Unable to configure the option.

## Steps to reproduce

> Nothing to reproduce.

## Proposed changes

> Add the configurable option to the configuration section in the Grocy addon.

**Comment by tringler** [comment]
@Kackerlacka I created a PR for that https://github.com/hassio-addons/addon-grocy/pull/151
